ITE/4/README.adoc

```diff
@@ -220,8 +219,8 @@ function get_hashable_representation(string generic_uri, kwargs) returns bytes \
         hashable_representation
 ```
 
-`hash_artifacts` closely mirrors the general workflow of hashing artifacts in
-the filesystem, and is similar to how the in-toto reference implementations
+`hash_artifacts` closely mirror the general workflow of hashing artifacts in
```

Comment:
`hash_artifacts` is the name of one of the functions that resolvers must implement. So while it ends in plural (for artifacts), it's actually a single entity.

Reply:
Why don't you make that comment a suggestion so I can just accept it here.

Reply:
Resolved. (I'm unable to mark it for some reason)

Reply:
OK. That looks good.

Reply:
Is there something I need to accept?

Reply:
No, this was taken care of last time with https://github.com/in-toto/ITE/pull/8/commits/6a606fee2ee66d5138a5243dea43ed88b45bdb4c and https://github.com/in-toto/ITE/pull/8/commits/5a401045ee242590896b50c46e697c709412dbd3. I was just going back and trying to mark the ones already fixed as resolved. Same for the other threads where I commented "Resolved".

Reply:
Ok. By all means remove/ignore any such comments.